source/app.d

```diff
@@ -8,23 +8,23 @@ struct CmdOptions
 	private string stdoutFile;
 	private string stderrFile;
 	private string scriptOnRestart;
-	private int max=-1;
-	private int minUptime=1000;
+	private int max = -1;
+	private int minUptime = 1000;
 
-    ///
-	public void parse(string[] _args)
+	///
+	public void parse(ref string[] _args)
```

Comment:
`getopt` condenses the array when it removes an arg, need `ref` so the shortening is visible in the caller